src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * The Duke program implements an application that stores and keeps track of tasks.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor to take in a specified filepath to create a Duke object.
+     *
+     * @param filePath the path location of the load or save file
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            tasks = new TaskList();
+        }
+    }
```

Comment:
I feel that a `if-else` statement to check whether the storage file exists is more suited here, rather than using `try-catch`.
As such, I might rewrite it as follows:
```java
File file = new File(filePath);
if (file.exists() && file.isReadable()) {
    tasks = new TaskList((new Storage(file)).load());
} else {
    tasks = new TaskList();
}
```
Of course, this means that your `Storage` class constructor should take in a `File` rather than `String` which I'll talk more about later.

```